__test__/integration/servicing_api.spec.ts

```diff
@@ -393,6 +399,30 @@ describe("Debt Servicing API (Integration Tests)", () => {
         });
     });
 
+    describe("#getDebtRegistryEntry", () => {
+        test("throws DOES_NOT_CONFORM_TO_SCHEMA error", async () => {
+            await expect(servicingApi.getDebtRegistryEntry(issuanceHash.substr(8))).rejects.toThrow(
+                /instance does not match pattern/,
+            );
+        });
+
+        test("returns the expected entry", async () => {
```

Comment:
Should include test case for when entry does not exist / should throw a human readable error when that happens.